action.yml

```diff
@@ -4,11 +4,15 @@ inputs:
   source:
     description: 'The source file'
     required: false
-    default: 'index.bs'
+    default: 'dom.bs'
```

Comment:
> Please revert this. We use `index.bs` for the default input filename because that is used in the [Bikeshed docs](https://tabatkins.github.io/bikeshed/).

Sorry, I forgot to restore defaults value